apps/web/src/server/db/migrate.ts

```diff
@@ -0,0 +1,22 @@
+import { sql } from "@vercel/postgres";
+import { config } from "dotenv";
```

Comment:
This seems to be unused, delete id if it's not needed.

Reply:
Accually it is used, look again.

Reply:
Look again, It accually is used.